src/data-mgt/node/package.json

```diff
@@ -13,7 +13,9 @@
         "stage-pc": "set NODE_ENV=staging&&nodemon ./bin",
         "prod-pc": "set NODE_ENV=production&&nodemon ./bin",
         "create-env": "printenv > .env",
-        "test": "mocha ./**/test/ut_*.js --exit"
+        "test": " mocha ./**/test/ut_*.js",
```

Comment:
@Psalmz777 , can you share some details around this configuration change., I am noticing different styles across the different PRs I have just merged. cc: @Kisakye-e

Reply:
@Baalmart , I used mocha with instanbul which is a code coverage tool for javascript

Reply:
@Baalmart I thought this would trigger the coverage checks initially before i noticed the token issue..I have changed it back though